landlab/components/mass_wasting_runout/mass_wasting_runout.py

```diff
@@ -0,0 +1,1346 @@
+      0.2037295   0.11279894  0.          0.10520981  0.14056859  0.12059567
+      0.18147989  0.12407022  0.1418186   0.19386482  0.13259837  0.23128465
+      0.08609032]
+
+    Also see that the attribute value is set to zero at any node in which the regolith
+    depth is 0.
+
+
+    References
+    ----------
+    Keck, J., Istanbulluoglu, E., Campforts, B., Tucker G., Horner-Devine A., (2023),
+    A landslide runout model for sediment transport, landscape evolution and hazard
+    assessment applications, submitted to Earth Surface Dynamics.
+
+    """
+
+    _name = "MassWastingRunout"
+
+    _unit_agnostic = False
+
+    _info = {
+        "mass__wasting_id": {
+            "dtype": int,
+            "intent": "in",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "interger or float id of each mass wasting area is assigned \
+                to all nodes representing the mass wasting area.",
+        },
+        "topographic__elevation": {
+            "dtype": float,
+            "intent": "in",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "elevation of the ground surface",
+        },
+        "soil__thickness": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "m",
+            "mapping": "node",
+            "doc": "regolith (soil) thickness, measured perpendicular to the \
+            land surface and includes all materials above the unweathered bedrock \
+            surface, e.g., saprolite, colluvium, alluvium, glacial drift",
+        },
+        "flow__receiver_node": {
+            "dtype": int,
+            "intent": "out",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "Node array of receivers (node that receives flow from current node)",
+        },
+        "flow__receiver_proportions": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "Node array of proportion of flow sent to each receiver.",
+        },
+        "topographic__steepest_slope": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "The steepest *downhill* slope",
+        },
+        "particle__diameter": {
+            "dtype": float,
+            "intent": "out",
+            "optional": True,
+            "units": "m",
+            "mapping": "node",
+            "doc": "representative particle diameter at each node, this might \
+            vary with underlying geology, contributing area or field observations",
+        },
+    }
+
+    def __init__(
+        self,
+        grid,
+        critical_slope,
+        threshold_flux,
+        erosion_coefficient,
```

Comment:
The only argument for the ``__init__`` method of *Landlab* components should be a *Landlab* grid. This means that ``critical_slope``, ``threshold_flux``, and ``erosion_coefficient`` should be keywords.